Libraries/LibDebug/DebugSession.h

```diff
@@ -104,14 +108,14 @@ class DebugSession {
 
 private:
     // x86 breakpoint instruction "int3"
-    static constexpr u8 BREAKPOINT_INSTRUCTION
-        = 0xcc;
+    static constexpr u8 BREAKPOINT_INSTRUCTION = 0xcc;
 
     int m_debugee_pid { -1 };
     bool m_is_debugee_dead { false };
 
     MappedFile m_executable;
-    ELF::Loader m_elf;
+    NonnullRefPtr<const ELF::Loader> m_elf;
```

Comment:
The fact that this didn't break immediately when you made `ELF::Loader` refcounted is why I like to add static construction helpers for ref-counted objects :)